core/src/main/web/app/mainapp/components/notebook/cell-directive.js

```diff
@@ -179,17 +179,20 @@
 
         $scope.cellview.menu.addItem({
           name: 'Delete cell',
+          shortcut: ['Ctrl-Alt-D', 'AltCmdD'],
```

Comment:
for mac, the shortcut strings should include a space or dash because this is hard to read.
but remove it in the replace() function.